charts/nexus-iq/tests/deployment_test.yaml

```diff
@@ -65,6 +65,17 @@ tests:
             - name: admin
               containerPort: 8071
               protocol: TCP
+      - equal:
+          path: spec.template.spec.containers[0].startupProbe
+          value:
+            httpGet:
+              path: /ping
+              port: admin
+            initialDelaySeconds: 30
+            failureThreshold: 30
```

Comment:
Large database upgrades takes more than 5 mins to complete. So we should have larger failureThreshold (180) 30 mins, to cater most scenarios. 

Having successThreshold: 1 would return quicker online status if the upgrade finishes before 30 mins.

Reply:
Updated [ebfa50b](https://github.com/sonatype/helm3-charts/pull/282/commits/ebfa50b5b87fb1700e5530e1b44a6e8ccb051b97)